api/src/components/crosslink/service.ts

```diff
@@ -84,8 +86,36 @@ export const exists = async (crosslinkId: string) => {
     return Boolean(await client.prisma.crosslink.findUnique({ where: { id: crosslinkId } }));
 };
 
-export const setVote = (crosslinkId: string, userId: string, vote: boolean) =>
-    client.prisma.crosslinkVote.upsert({
+const updateScore = async (crosslinkId: string) => {
+    const crosslink = await client.prisma.crosslink.findUnique({
+        where: {
+            id: crosslinkId
+        },
+        select: {
+            votes: {
+                select: {
+                    vote: true
+                }
+            }
+        }
+    });
+
+    const score = crosslink?.votes
```

Comment:
Handling crosslink being null?

Reply:
Yeah, that's missing. Just added a bit to return false if it is null.